src/metabase/upload.clj

```diff
@@ -104,12 +139,24 @@
   "All type tags that correspond to concrete column types."
   (into #{} (remove abstract->concrete) value-types))
 
-(defn ^:private column-type
-  "The most specific column type corresponding to the given value type."
+(defn- column-type?
   [value-type]
-  (or (abstract->concrete value-type value-type)
-      ;; If we know nothing about the value type, treat it as an arbitrary string.
-      ::text))
+  (contains? column-types value-type))
+
+(defn ^:private column-type
```

Comment:
I keep thinking of this function as the "concretize" function so maybe we should rename it? You even used that word in our 1:1